python/kvikio/tests/test_examples.py

```diff
@@ -26,3 +26,10 @@ def test_zarr_cupy_nvcomp(tmp_path, monkeypatch):
 
     monkeypatch.syspath_prepend(str(examples_path))
     import_module("zarr_cupy_nvcomp").main(tmp_path / "test-file")
+
+
+def test_http_io(tmp_path, monkeypatch):
+    """Test examples/http_io.py"""
+
+    monkeypatch.syspath_prepend(str(examples_path))
+    import_module("http_io").main(tmp_path)
```

Comment:
Does this need to skip the test if remote file support is not available?

Reply:
Good catch, fixed